DietPi-Software | PHP: Fix 'upload_tmp_dir' issue

MichaIng opened the pull request:
- https://github.com/Fourdee/DietPi/issues/1144
+ reorder installation to use own ini file.

Commit:
DietPi-Software | PHP: Fix 'upload_tmp_dir' issue

+ reorder installation to use own ini file.

Fourdee commented on dietpi/dietpi-software:
```diff
@@ -8832,15 +8832,18 @@ _EOF_
 
-				echo -e "nothing here" &> /dev/null
+				# https://github.com/Fourdee/DietPi/issues/1144
+				local php_service=/lib/systemd/system/apache2.service
```
Please define as characters/string eg:
```local php_service='/lib/systemd/system/apache2.service'```

Prevents any leakage to bash.

MichaIng replied:
Yes can do this here. But for `/lib/systemd/system/php*-fpm.service` it does not work, as `*` is then not interpreted as optional `5` or `7.0` dependant on PHP version. Tried with `'` and `"`...
Tested on Jessie and Stretch, and without string definition it works fine. `bash -x testscript` looks like it is in the end interpreted as string but replaces `*` correctly.

Do you think it is okay to leave it as path for second statement, or might there be other problems with it? Of course I can also just do `$DISTRO` check and then define it as fixed string.

Fourdee replied:
@MichaIng 

```
root@DietPi:~# test='/root/test*'
root@DietPi:~# echo 1 > test3
root@DietPi:~# echo 1 > test2
root@DietPi:~# echo 1 > test1
root@DietPi:~# ls -lha | grep test
-rw-r--r--  1 root root    2 Nov 27 17:43 test1
-rw-r--r--  1 root root    2 Nov 27 17:43 test2
-rw-r--r--  1 root root    2 Nov 27 17:43 test3
root@DietPi:~# rm "$test"
rm: cannot remove '/root/test*': No such file or directory
root@DietPi:~# rm $test
root@DietPi:~# ls -lha | grep test
root@DietPi:~#
```

```
php_service='/lib/systemd/system/php*-fpm.service'
```
Should work using:
```
 $php_service
```

MichaIng replied:
@Fourdee 
Ah yes, I tested with `test='~/test*'` and there it is not working, seems to be not interpreted as path with possible variables `*` anymore. But beginning with `/root` instead works, so will also work in this script. Adjustment done

Commit:
DietPi-Software | PHP: Fix 'upload_tmp_dir' on update

Commit:
DietPi-Software | PHP: Add removal of 'dietpi.ini' to uninstallation

+ Remove added php config files before AGP, to allow apt removing the base folders.

Commit:
DietPi-Software | PHP: Set file path correctly as string

Commit:
DietPi-Software | PHP: Correctly set file path as string variable

Commit:
v159

Commit:
Merge branch 'testing' into patch-1

Commit:
Merge branch 'testing' into patch-1

Fourdee commented:
@MichaIng 

Legend 👍.
